packages/apps/human-app/server/src/common/interceptors/interceptor.module.ts

```diff
@@ -1,9 +1,21 @@
 import { Module, Global } from '@nestjs/common';
 import { AxiosRequestInterceptor } from './axios-request.interceptor';
+import { EnvironmentConfigService } from '../config/environment-config.service';
 
 @Global()
 @Module({
-  providers: [AxiosRequestInterceptor],
+  providers: [
+    {
+      provide: AxiosRequestInterceptor,
+      useFactory: (configService: EnvironmentConfigService) => {
+        if (configService.axiosRequestLoggingEnabled) {
```

Comment:
Consider using configurable logging levels (e.g., debug) from environment settings for a more flexible approach